inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -785,6 +785,15 @@ private function add_used_fonts_preload( string $html, string $used_css ): strin
 			return $html;
 		}
 
+		/**
+		 * Filters the list of fonts to exclude from preload
+		 *
+		 * @since 3.15.10
+		 *
+		 * @param array $excluded_fonts_preload List of fonts to exclude from preload
+		 */
+		$exclude_fonts_preload = apply_filters( 'rocket_exclude_rucss_fonts_preload', [] );
+
```

Comment:
We should add a safe guard in case the filter value is not an array, to fallback to the default value, like we do in other places

Reply:
I've added a safeguard :)